addons/mail/static/src/components/thread_preview/thread_preview.xml

```diff
@@ -39,6 +39,10 @@
                                 <t t-esc="thread.lastMessage.date.fromNow()"/>
                             </span>
                         </t>
+                        <t t-if="thread and thread.rtcSessions.length">
+                            <span class="o-autogrow"/>
```

Comment:
I would not move the date on the middle, this is really strange. I would suggest to move the icon somewhere else, probably just before the existing `o-autogrow`?